### PR TITLE
ITN P2

### PR DESCRIPTION
**Pull request by YazzyYaz**

### CODE REVIEW

File: docs/nodes/blockspace-race.md [hunk 1]
```diff
@@ -229,6 +229,65 @@ The default port is 1317.
...
+
+* Start: 28 Mar 2023
+* End: 8 May 2023
+
+<details>
+<summary>Validator Tasks</summary>
+
+| Validator Tasks | Task type | Points |
+| ------------- | ------------- | ------------- |
+| [Write a technical article](./itn-setup-writeup.md) | Bonus  | Bonus Pool |
```
Comment (jcstein) [inline docs/nodes/blockspace-race.md:273]:
```suggestion
| [Write a technical article](./itn-setup-writeup.md) | Bonus  | Bonus pool |
```

File: docs/nodes/blockspace-race.md [hunk 2]
```diff
@@ -229,6 +229,65 @@ The default port is 1317.
...
+| [Create Toolings for the Celestia Network](./itn-toolings.md) | Bonus  | Bonus Pool |
+| [Run a public seed or RPC node](./itn-rpc-node.md) | Bonus | Bonus Pool |
```
Comment (jcstein) [inline docs/nodes/blockspace-race.md:275]:
```suggestion
| [Run a public seed or RPC node](./itn-rpc-node.md) | Bonus | Bonus pool |
```

File: docs/nodes/blockspace-race.md [hunk 3]
```diff
@@ -229,6 +229,65 @@ The default port is 1317.
...
+| [Deploy a sovereign rollup](./itn-deploy-rollup.md) | Bonus  | Bonus Pool |
+</details>
+
+<details>
+<summary>Node Operator Tasks</summary>
+
+| Node Operator Tasks | Task type | Points |
+| ------------- | ------------- | ------------- |
+| [Write a technical article](./itn-setup-writeup.md) (bridge and full storage nodes) | Bonus  | Bonus Pool |
```
Comment (jcstein) [inline docs/nodes/blockspace-race.md:284]:
```suggestion
| [Write a technical article](./itn-setup-writeup.md) (bridge and full storage nodes) | Bonus  | Bonus pool |
```

File: docs/nodes/itn-claim-rewards.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+Claim your validator rewards for producing blocks as part of
+Celestia's Incentivized Testnet validator set. You can learn
+about how to claim validator rewards in the tutorial [here](https://docs.celestia.org/nodes/celestia-app-commands#claim-validator-rewards).
+
+## Directions
+
+1. Claim your validator rewards for your validator.
+2. Keep a record of the transaction for the validator reward you claimed.
+
+## Judging Criteria
```
Comment (jcstein) [inline docs/nodes/itn-claim-rewards.md:14]:
```suggestion
## Judging criteria
```

File: docs/nodes/itn-claim-rewards.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+1. Full points if you claim your validator rewards.
+2. No points if you do not.
+
+## How to Submit
```
Comment (jcstein) [inline docs/nodes/itn-claim-rewards.md:19]:
```suggestion
## How to submit
```

File: docs/nodes/itn-deploy-bridge.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
+# Deploy A Bridge Node
```
Comment (jcstein) [inline docs/nodes/itn-deploy-bridge.md:1]:
```suggestion
# Deploy a bridge node
```

File: docs/nodes/itn-deploy-light.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
...
+3. How to find your light node id is found [here](https://docs.celestia.org/developers/node-api/#post-p2pinfo).
+4. Make sure that the wallet address the node uses is funded.
+5. The keys directory under `.celestia-light-blockspacerace-0/keys`
+  contains your `pubkey` for both DA and Tendermint networks. In
+  the leaderboard your pubkey for DA nodes looks similar to this
+  `12D3KooWSJoAF9pF7AkyEAhQCtNF9TA8JrAujWdqoPJ11uojuKZb`
+
+Please do a backup of the keys directory in order to not lose your progress.
+
+## Judging Criteria
```
Comment (jcstein) [inline docs/nodes/itn-deploy-light.md:26]:
```suggestion
## Judging criteria
```

File: docs/nodes/itn-node-analysis.md [hunk 8]
```diff
@@ -0,0 +1,30 @@
+# Perform Analysis of Your Node
```
Comment (jcstein) [inline docs/nodes/itn-node-analysis.md:1]:
```suggestion
# Perform analysis of your node
```

File: docs/nodes/itn-node-analysis.md [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+2. Ensure to share any code scripts you used to perform your
+  analysis in your blog post.
+
+## Judging Criteria
+
+1. Full points if you submit a clear, concise write up with useful analysis
+  of your node.
+2. No points if you do not.
+
+## How to Submit
```
Comment (jcstein) [inline docs/nodes/itn-node-analysis.md:26]:
```suggestion
## How to submit
```

File: docs/nodes/itn-pfb-ui.md [hunk 10]
```diff
@@ -0,0 +1,25 @@
+# Create a UI for Submitting PayForBlob Transactions
```
Comment (jcstein) [inline docs/nodes/itn-pfb-ui.md:1]:
```suggestion
# Create a UI for submitting PayForBlob transactions
```

File: docs/nodes/itn-pfb-ui.md [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+You can check out the Node tutorial [here](https://docs.celestia.org/developers/node-tutorial).
+It shows you how you can call the API in order to submit a PFB transaction.
+
+## Directions
+
+1. Submission is limited to only 1, do not submit more than one.
+2. Create a UI for submitting a PFB transaction.
+3. Ensure it is open source and share the Github repository as well.
+
+## Judging Criteria
```
Comment (jcstein) [inline docs/nodes/itn-pfb-ui.md:15]:
```suggestion
## Judging criteria
```

File: docs/nodes/itn-rpc-node.md [hunk 12]
```diff
@@ -0,0 +1,35 @@
+# Run a Public RPC Node
```
Comment (jcstein) [inline docs/nodes/itn-rpc-node.md:1]:
```suggestion
# Run a public RPC node
```

File: docs/nodes/itn-rpc-node.md [hunk 13]
```diff
@@ -0,0 +1,35 @@
...
+## Directions
+
+1. Run a publicly-available RPC node by following the
+  Consensus Full Node tutorial.
+2. Provide the IP and port of the available RPC node to the community
+  and ensure the correct ports are open so that the community
+  can use it.
+3. Please submit a PR to our docs with your RPC [here](https://github.com/celestiaorg/docs/blob/main/docs/nodes/blockspace-race.md#rpc-endpoints).
+
+## Judging Criteria
```
Comment (jcstein) [inline docs/nodes/itn-rpc-node.md:18]:
```suggestion
## Judging criteria
```

File: docs/nodes/itn-rpc-node.md [hunk 14]
```diff
@@ -0,0 +1,35 @@
...
+
+1. Full points if your RPC node is available for use by others and Data Availability
+  nodes can connect to it.
+2. No points if not.
+
+## How to Submit
```
Comment (jcstein) [inline docs/nodes/itn-rpc-node.md:24]:
```suggestion
## How to submit
```

File: docs/nodes/itn-toolings.md [hunk 15]
```diff
@@ -0,0 +1,37 @@
+# Create Toolings for the Celestia Network
```
Comment (jcstein) [inline docs/nodes/itn-toolings.md:1]:
```suggestion
# Create toolings for the Celestia Network
```

File: docs/nodes/itn-toolings.md [hunk 16]
```diff
@@ -0,0 +1,37 @@
...
+
+## Directions
+
+1. Submission is limited to only 1, do not submit more than one.
+2. Build a tool that can help strengthens the validator community,
+  whether it is an analytics tool, a dev-tool, a command-line script,
+  or a useful dashboard.
+3. The tool should have supporting documentation or readme.
+
+## Judging Criteria
```
Comment (jcstein) [inline docs/nodes/itn-toolings.md:24]:
```suggestion
## Judging criteria
```

**Review by jcstein — CHANGES_REQUESTED** [review]
added some changes in sentence case

**Review by jcstein — CHANGES_REQUESTED** [review]
added some changes in sentence case

File: docs/nodes/itn-enable-telemetry-nodes.md [hunk 17]
```diff
@@ -0,0 +1,33 @@
...
+1. Add `--metrics.tls=false --metrics --metrics.endpoint http://otel.celestia.tools:4318`
+  to your node's start command and restart your node.
+
+## Judging Criteria
+
+1. Full points if we can see data from your Celestia Node
+2. No points if we cannot
+3. Note that we will keep track of your metrics throughout the whole
+  ITN so make sure you leave it connected throughout all phases.
+
```
Comment (mindstyle85) [inline docs/nodes/itn-enable-telemetry-nodes.md:22]: I would add point 4 here:

4. Make sure you can see your node ID on [https://tiascan.com](https://tiascan.com/) in your respective node category. You can also check your node uptime status there.

**Review by jcstein — APPROVED** [review]

**Pull request merged**